packages/client/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Should this be here?

Reply:
There were `LICENSE` in some of the packages earlier so I just added it to all of them. I also think it's enough to have a single one in root if @KATT is fine with that too?